Sources/MongoSwift/MongoConnectionString.swift

```diff
@@ -742,12 +743,112 @@ public struct MongoConnectionString: Codable, LosslessStringConvertible {
         try? self.init(throwsIfInvalid: description)
     }
 
-    // TODO: SWIFT-1405: add options to description
     public var description: String {
```

Comment:
Would it be un-idiomatic to also include a `toString()` method for greater discoverability? I know I've asked this in the past but I can't remember the conclusion we came to.

Reply:
I think it is considered un-idiomatic, none of the standard library/Foundation types that would make sense to have one for (like URL, Date, any of the number types...) have them, and none of the protocols relevant to types being able to provide text representation of themselves include such methods.

I think generally, types should conform to relevant String protocols, and you can either use `String(describing:)`, `String(reflecting:)`, or `description` directly to get the description based on use case. interestingly it's generally not advised to call `description` directly except for `LosslessStringConvertible` types (which `MongoConnectionString` is) - see https://oleb.net/2018/dont-call-description/